ci/checksizes.sh

```diff
@@ -45,11 +45,11 @@ case $FULLVER in
     REVERSI_PREOPT_GZ_EXPECTEDSIZE=71000
     REVERSI_OPT_GZ_EXPECTEDSIZE=32000
     ;;
-  2.12.0-M5)
-    REVERSI_PREOPT_EXPECTEDSIZE=528000
-    REVERSI_OPT_EXPECTEDSIZE=124000
-    REVERSI_PREOPT_GZ_EXPECTEDSIZE=67000
-    REVERSI_OPT_GZ_EXPECTEDSIZE=30000
+  2.12.0-RC1)
+    REVERSI_PREOPT_EXPECTEDSIZE=595000
```

Comment:
Define "good" :P The reason is https://github.com/scala-js/scala-js/pull/2590#issuecomment-246685424
It's coming from scala/scala, there's nothing we can do about right now (in this PR), but I've recorded the problem as #2591 so that we can address it later.